packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -155,6 +156,29 @@ class FlatTableState<T> extends State<FlatTable<T>>
   }
 }
 
+class Selection<T> {
+  Selection(
+      {this.node,
+      this.nodeIndex,
+      this.scrollIntoView = false,
+      this.displayRow});
+
+  Selection.empty()
+      : node = null,
+        nodeIndex = null,
+        scrollIntoView = false,
+        displayRow = null;
+
+  final T node;
+  final int nodeIndex;
+  final bool scrollIntoView;
+  int displayRow;
```

Comment:
how is displayRow different from nodeIndex?

Reply:
there is no difference nodeIndex is used.